### PR TITLE
[CloudPC]add singleSignOn into ProvisioningPolicy and OrganizationSetting

### PR DESCRIPTION
**Pull request by Guoan-Tang**

### CODE REVIEW

File: api-reference/beta/resources/cloudpcorganizationsettings.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Represents the Cloud PC organization settings for a tenant. A tenant has only on
 |Property|Type|Description|
 |:---|:---|:---|
 |enableMEMAutoEnroll|Boolean|Specifies whether new Cloud PCs will be automatically enrolled in Microsoft Endpoint Manager(MEM). The default value is `false`.|
+|enableSingleSignOn|Boolean|True if the provisioned Cloud PC can be accessed by single sign-on option, false if the provisioned Cloud PC don't support this feature. Default value is false. Through single sign-on, Windows 365 users can authenticate to Azure AD with passwordless options (like FIDO keys) to access their Cloud PC.|No|No|No|
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpcorganizationsettings.md:28]: Please update all description to: 
```suggestion
|enableSingleSignOn|Boolean|Describle......When TRUE, indicate...When FALSE, indicate... Default value is.......Through single sign-on, Windows 365 users can authenticate to Azure AD with passwordless options (like FIDO keys) to access their Cloud PC.|No|No|No|
```

**Comment by AshleyYangSZ** [comment]
@Guoan-Tang Please add change-log which is required for schema PR.

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@Guoan-Tang  please 
1.create roadmap workitem and attach the workitem number in comment.
2.Update the changelog to describe all your changes.

**Comment by Guoan-Tang** [comment]
roadmap workitem: 15771

**Review by RuiHou105 — APPROVED** [review]

File: api-reference/beta/api/cloudpcorganizationsettings-get.md [hunk 2]
```diff
@@ -105,6 +105,7 @@ Content-Type: application/json
   "value": {
     "@odata.type": "#microsoft.graph.cloudPcOrganizationSettings",
     "enableMEMAutoEnroll": false,
+    "enableSingleSignOn": "true",
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpcorganizationsettings-get.md:108]: It's a boolean, not string
```suggestion
    "enableSingleSignOn": true,
```

File: api-reference/beta/api/cloudpcprovisioningpolicy-get.md [hunk 3]
```diff
@@ -125,6 +125,7 @@ Content-Type: application/json
         "domainJoinType": "hybridAzureADJoin",
         "onPremisesConnectionId": "16ee6c71-fc10-438b-88ac-daa1ccafffff"
     },
+    "enableSingleSignOn": "true",
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpcprovisioningpolicy-get.md:128]: It's a boolean, not string

```suggestion
    "enableSingleSignOn": true,
```

File: api-reference/beta/api/cloudpcprovisioningpolicy-get.md [hunk 4]
```diff
@@ -206,6 +207,7 @@ Content-Type: application/json
         "regionName": null,
         "type": "hybridAzureADJoin"
     },
+    "enableSingleSignOn": "true",
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpcprovisioningpolicy-get.md:210]:
```suggestion
    "enableSingleSignOn": true,
```

File: api-reference/beta/api/cloudpcprovisioningpolicy-get.md [hunk 5]
```diff
@@ -302,6 +304,7 @@ Content-Type: application/json
...
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpcprovisioningpolicy-get.md:307]:
```suggestion
    "enableSingleSignOn": true,
```

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
Please resolve all remaining comments.

**Review by RuiHou105 — APPROVED** [review]

**Comment by JarbasHorst** [comment]
@Lauragra, this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**